aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/ListHandler.java

```diff
@@ -13,6 +17,14 @@
 
 public class ListHandler extends BaseHandlerStd {
 
+    final static Collection<String> CUSTOM_ENGINES = ImmutableList.of(
```

Comment:
Have you considered client side filter for custom engine versions? CFN in that case will not need to be updated when new custom engine is added.

Reply:
- If we implement client-side filtering, it means that we will be sending empty intermediate pages back to the ListHandler caller. Please note that the ListHandler only returns one page per invocation, not all of the collected data. 
- DescribeEngineVersions call is quite intensive in terms of response size and latency. It would be better to implement server-side filtering to avoid this.